modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -64,6 +64,185 @@ const goToWithReferrer = async function (page, referrer, url) {
 	}, url);
 };
 
+test('shows individuals who viewed a page less than 24 hours ago', async ({
+	apiHelpers,
+	page,
+}) => {
+	const projects = await apiHelpers.jsonWebServicesOSBFaro.getProjects();
+
+	const project = projects.find(({name}) => name === 'FARO-DEV-liferay');
+
+	const channel = await apiHelpers.jsonWebServicesOSBFaro.createChannel(
+		'My Property - ' + getRandomString(),
+		project.groupId
+	);
+
+	await apiHelpers.jsonWebServicesOSBAsah.createIndividuals([
+		{
+			emailAddress: 'user1@liferay.com',
+			fields: [
+				{
+					dataSourceId: 0,
+					name: 'givenName',
+					value: 'user1',
+				},
+				{
+					dataSourceId: 0,
+					name: 'familyName',
+					value: 'user1',
+				},
+				{
+					dataSourceId: 0,
+					name: 'email',
+					value: 'user1@liferay.com',
+				},
+			],
+			firstName: 'user1',
+			id: 'user1@liferay.com',
+			lastName: 'user1',
+		},
+		{
+			emailAddress: 'user1@liferay.com',
```

Comment:
Hey @marcosapmf, I think the correct here is user2@liferay.com, right?
```suggestion
			emailAddress: 'user2@liferay.com',
```